### PR TITLE
Fix encode json loads for python 3

### PR DESCRIPTION
**Pull request by pgferretti**
Fix python 3
TypeError: the JSON object must be str, not 'bytes'

### CODE REVIEW

File: pagarme/pagarme.py [hunk 1]
```diff
@@ -136,8 +136,8 @@ def get_all_resources(self, Class, page, count):
         url = Class.BASE_URL
         pagarme_response = requests.get(url, params=data)
         if pagarme_response.status_code != 200:
-            self.error(pagarme_response.content)
-        responses = json.loads(pagarme_response.content)
+            self.error(pagarme_response.content)
```
Comment (mbodock) [inline pagarme/pagarme.py]: trailing spaces.

File: pagarme/transaction.py [hunk 2]
```diff
@@ -36,11 +37,13 @@ def __init__(
         self.id = None
         self.data = {}
         self.customer = customer
+        self.split_rules = split_rules
 
         for key, value in kwargs.items():
             self.data[key] = value
 
     def error(self, response):
+        print(response)
```
Comment (mbodock) [inline pagarme/transaction.py]: Unnecessary print.

File: pagarme/transaction.py [hunk 3]
```diff
@@ -87,7 +98,7 @@ def __dict__(self):
             d['installments'] = self.installments
             d['payment_method'] = self.payment_method
             d['soft_descriptor'] = self.soft_descriptor[:13]
-
+
```
Comment (mbodock) [inline pagarme/transaction.py]: Trailing spaces.

File: pagarme/transaction.py [hunk 4]
```diff
@@ -98,14 +109,19 @@ def __dict__(self):
 
         if self.customer:
             d.update(self.customer.get_anti_fraud_data())
-
+
```
Comment (mbodock) [inline pagarme/transaction.py]: Trailing spaces

File: pagarme/transaction.py [hunk 5]
```diff
@@ -116,6 +132,17 @@ def refund(self):
...
+
+    def find_split_rule(self, id=None):
+        url = self.BASE_URL + '/' + str(id) + '/split_rules'
+        pagarme_response = requests.get(url, data=self.get_data())
+        if pagarme_response.status_code == 200:
+            return json.loads(pagarme_response.content.decode(encoding='UTF-8'))
         else:
             self.error(pagarme_response.content)
+
```
Comment (mbodock) [inline pagarme/transaction.py]: Trailing spaces.

**Comment by mbodock** [comment]
Hello @pgferretti  can you provide some information about this PullRequest.
Also try to fix the failing tests.

Thank you.

**Comment by lerrua** [comment]
@pgferretti any news about your PR?

**Comment by mbodock** [comment]
Well I'm closing this PR for lack of information.

**Pull request closed without merging**